requirements_dev.txt

```diff
@@ -0,0 +1,12 @@
+pip==9.0.1
```

Comment:
Just to make sure, do you have Pipfile and this requirements_dev.txt on purpose?  I am guess this is for rtd maybe?